plugin.js

```diff
@@ -167,7 +172,12 @@ class Plugin {
   }
 
   static processPostBuildPlugins(exp_info, options) {
-    const pluginConfig = require(Plugin.getConfigFileName(options.env));
+    let pluginConfig = require(Plugin.getConfigFileName(options.env));
+
+    if(!options.isValidate)
+    {
+      pluginConfig = pluginConfig.filter((p) => p.id !== "build-validation");

```

Comment:
Should the id change to tool-validation?

Reply:
Yeah, we have changed it to tool-validation